set.mm

```diff
@@ -52644,6 +52670,44 @@ Contrast with domain (defined in ~ df-dm ).  For alternate definitions,
       DUBUCLUAMAUBUCUANABCOPRST $.
   $}
 
+  ${
+    $d X x y $.  $d ps x $.
+    dmopabel.d $e |- ( x = X -> ( ph <-> ps ) ) $.
+    $( A set is an element of the domain of a ordered pair class abstraction
+       iff there is a second set so that both sets fulfil the wff of the class
+       abstraction.  (Contributed by AV, 19-Oct-2023.) $)
+    dmopabelb $p |- ( X e. V -> ( X e. dom { <. x , y >. | ph }
+                                  <-> E. y ps ) ) $=
+      ( copab cdm wcel wex cab dmopab eleq2i cv wceq exbidv eqid elab2g syl5bb
+      ) FACDHIZJFADKZCLZJFEJBDKZUAUCFACDMNUBUDCFUCECOFPABDGQUCRST $.
+  $}
+
+  ${
+    $d A x y z $.  $d B i x y z $.  $d C x y z $.  $d D x y z $.  $d I x y z $.
+    $d U i x y z $.  $d V i x y z $.  $d W z $.  $d X i z $.  $d i u x y z $.
+    $d i v x y z $.
+    $( The domain of an ordered pair class abstraction with two nested
+       restricted existential quantifiers.  (Contributed by AV,
+       23-Oct-2023.) $)
+    dmopab2rex $p |- ( A. u e. U ( A. v e. V B e. X /\ A. i e. I D e. W )
+                       -> dom { <. x , y >. | E. u e. U ( E. v e. V
+                         ( x = A /\ y = B ) \/ E. i e. I ( x = C /\ y = D ) ) }
+                = { x | E. u e. U ( E. v e. V x = A \/ E. i e. I x = C ) } ) $=
```

Comment:
I don't have issues with the other theorems added to main, but this one seems to be too specific. I suppose you don't want to add such generic theorems directly to @digama0 's mathbox?

Reply:
I also was torn between putting it into main and @digama0 's mathbox (just before ~satffunlem1lem2, where it is used), but finally I thought it is kind of generally interesting (how to cope with existential quantifiers in an ordered pair abstraction), so I have put it into main. If there is no agreement to have it in main, I can move it into MC's mathbox.

~ dmopab3rexdif is a similar theorem, but this is really too specific and should be regarded as lemma for ~satffunlem2lem2.